app/controllers/groceries_controller.rb

```diff
@@ -72,6 +72,10 @@ def email_group
     redirect_to @grocery.user_group, notice: 'All group members have been emailed the grocery list.'
   end
 
+  def recipes
+    render json: JSON.parse(Nokogiri::HTML(open("http://food2fork.com/api/search?key=#{ENV["FOOD2FORK_KEY"]}&q=#{@grocery.items.map(&:name).join(",")}")))
```

Comment:
Line is too long. [154/80]<br>Prefer single-quoted strings inside interpolations.